Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -135,14 +135,17 @@
               <button mat-button [routerLink]="['/rooms']" i18n="@@navigationRooms">
                 Rooms
               </button>
-              <button mat-icon-button [matMenuTriggerFor]="personalmenu">
+              <button mat-icon-button [matMenuTriggerFor]="personalmenu" class="profilebutton">
                 <mat-icon
+                  *ngIf="currentUser == null"
                   matBadge="!"
                   matBadgeColor="accent"
                   matBadgeSize="small"
                   [matBadgeHidden]="!releaseInfoService.newUpdate"
                   >person</mat-icon
                 >
+                <app-profile-picture *ngIf="currentUser != null" class="pb-self-centered circle square-img40" [user]="currentUser" [altText]="currentUser.Id">

```

Comment:
Please use ` [imageSize]="50"` so we can save some bandwidth. :)

Reply:
ok, i fixed this in the newest commit